.github/workflows/create-wheels.yaml

```diff
@@ -281,12 +302,13 @@ jobs:
           twine upload --skip-existing dist/*manylinux*
 
   make-wheel-linux-arm64:
```

Comment:
this was probably ok as it was, since it's using a docker images, ubuntu is just used to run docker, but it should work either way